docs/content/users/quickstart.md

```diff
@@ -153,30 +153,59 @@ While the generic `php` project type is [ready to go](./project.md) with any CMS
 
     ## Laravel
 
-    Use a new or existing Composer project, or clone a Git repository.
-
-    The Laravel project type can be used for [Lumen](https://lumen.laravel.com/) just as it can for Laravel. DDEV automatically updates or creates the .env file with the database information.
-
-    === "Composer"
-        ```bash
-        mkdir my-laravel-app
-        cd my-laravel-app
-        ddev config --project-type=laravel --docroot=public --create-docroot
-        ddev composer create --prefer-dist --no-install --no-scripts laravel/laravel
-        ddev composer install
-        ddev exec "php artisan key:generate"
-        ddev launch
-        ```
-    === "Git Clone"
-        ```bash
-        git clone <your-laravel-repo>
-        cd <your-laravel-project>
-        ddev config --project-type=laravel --docroot=public --create-docroot
-        ddev start
-        ddev composer install
-        ddev exec "php artisan key:generate"
-        ddev launch
-        ```
+    === "Laravel"
+
+        Use a new or existing Composer project, or clone a Git repository.
+
+        The Laravel project type can be used for [Lumen](https://lumen.laravel.com/) just as it can for Laravel. DDEV automatically updates or creates the .env file with the database information.
```

Comment:
Wrap `.env` in backticks:

```suggestion
        The Laravel project type can be used for [Lumen](https://lumen.laravel.com/) just as it can for Laravel. DDEV automatically updates or creates the `.env` file with the database information.
```